File: 코딩테스트 입문/Javascript/배열 비교하기.js
function solution(arr1, arr2) {
    var answer = 0;
    
    if(arr1.length>arr2.length){
        answer = 1;
    }
    else if(arr1.length<arr2.length){
        answer = -1;
    }
    else{
        let arr1_sum = arr1.reduce((a,c)=>a+c);
        let arr2_sum = arr2.reduce((a,c)=>a+c);
        
        if(arr1_sum>arr2_sum){
            answer = 1;
        }
        else if(arr1_sum<arr2_sum){
            answer = -1;
        }
        else{
            answer = 0;
        }
    }
    
    return answer;
}

File: 코딩테스트 입문/Python/배열 비교하기.py
def solution(arr1, arr2):
    answer = 0
    arr1Sum = sum(arr1)
    arr2Sum = sum(arr2)
    
    if len(arr1) == len(arr2):
        if arr1Sum == arr2Sum:
            answer = 0
        elif arr1Sum > arr2Sum:
            answer = 1
        else:
            answer = -1
    else:
        if len(arr1) > len(arr2):
            answer = 1
        else:
            answer = -1
                
    return answer